docs/changelog.md

```diff
@@ -61,7 +61,17 @@ Legend:
 > 4. Retention – Configure how long you want to keep Cost Management exports and normalized data in storage.
 > 5. Analytics engine – Ingest cost data into an Azure Data Explorer cluster.
 
-<br>
+📒 Azure Monitor workbooks
+{: .fs-5 .fw-500 .mt-4 mb-0 }
+
+> ➕ Added:
+>
+> 1. Respect global subscription filter selections
+
+> 🛠️ Fixed:
+>
+> 1. Fixed a bug with Azure Hybrid benefits.
+> 2. Fixed a bug with the SQL Managed Instance query.
```

Comment:
Can you explain what the bugs are so people know if they should prioritize updating? If there's an existing issue, you can link to them. (See examples below.)

Reply:
@arthurclares can you help with a description of the changes ?